src/main/java/Duke.java

```diff
@@ -1,10 +1,233 @@
+                    break;
+                case "E":
+                    newTask = new Event(command[2], command[3], command[4]);
+                    break;
+                default:
+                    throw new InvalidCommandException("");
+                }
+                if (command[1].equals("X")) {
+                    newTask.markAsDone();
+                }
+                current_list.add(newTask);
+            }
+            sc.close();
+        } catch (FileNotFoundException e) {
+            System.out.println(e.toString());
+        }
+
+    }
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+        try {
+            Duke.loadFile();
+        } catch (Exception e) {
+            System.out.println(e);
+            return;
+        }
+
+        Scanner user_input = new Scanner(System.in);
+
+        System.out.println("Hello! I'm Anton's Bot");
+        System.out.println("What can I do for you?");
+
+        while (true) {
+            try {
+                // Handling Input
+                String current_input = user_input.nextLine();
+                System.out.println(PARTITION);
+                String[] current_input_array = current_input.split(" ", 2);
+                String input_command = current_input_array[0];
+
+                // Handling Various Commnds
+                if (input_command.equals(EXIT_COMMAND)) {
+                    handleExit();
+                    System.out.println(PARTITION);
+                    break;
+                } else if (input_command.equals(LIST_COMMAND)) {
+                    handleList();
+                } else if (input_command.equals(DELETE_COMMAND)) {
+                    handleDelete(current_input_array);
+                } else if (input_command.equals(MARK_COMMAND)) {
+                    handleMark(current_input_array);
+                } else if (input_command.equals(UNMARK_COMMAND)) {
+                    handleUnmark(current_input_array);
+                } else if (input_command.equals(TODO_COMMAND)) {
+                    handleTodo(current_input_array);
+                } else if (input_command.equals(EVENT_COMMAND)) {
+                    handleEvent(current_input_array);
+                } else if (input_command.equals(DEADLINE_COMMAND)) {
+                    handleDeadline(current_input_array);
+                } else {
+                    throw new InvalidCommandException("");
+                }
+                System.out.println(PARTITION);
+            } catch (Exception e) {
+                // handle exception
+                System.out.println(e.getMessage());
+                System.out.println(PARTITION);
```

Comment:
Would adding the `System.out.println(PARTITION)` lines to a new `finally` block look better?